typings/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.base.json",
+  "compilerOptions": {
+    "skipLibCheck": false,
```

Comment:
enabling lib checking - a proper way to make sure custom ambient types are valid